shared/css/course-blocks.scss

```diff
@@ -268,82 +228,6 @@
     padding-right: 10px;
   }
 
-  .audiencebar {
-    width: 100%;
-    padding: 4px 30px;
-    margin: 0;
-    bottom: 25px;
-    float: left;
-    position: absolute;
-  }
-
-  .bottombar {
-    width: 100%;
-    padding-left: 30px;
-    padding-right: 10px;
-    padding-top: 4px;
-    padding-bottom:4px;
-    margin: 0;
-    bottom: 0;
-    float: left;
-    position: absolute;
-
-    html[dir=rtl] & {
```

Comment:
Do these cards still work in RTL?

Reply:
these styles weren't being used, so deleting this won't have any effect on how the tiles work in RTL. i think i'll need to make some alignment tweaks to get these tiles working in RTL, so i'll do that in a follow-up